code/modules/client/preference_setup/general/02_body.dm

```diff
@@ -2,7 +2,7 @@ var/global/list/valid_bloodtypes = list("A+", "A-", "B+", "B-", "AB+", "AB-", "O
 
 /datum/preferences
 	var/species = SPECIES_HUMAN
-	var/b_type = "A+"					//blood type (not-chooseable)
```

Comment:
you could note that you made the default blood type O- instead of A+ in the changelog but thats a minor nitpick